server/services/ml_enabler_service.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import asyncio
+import mercantile
+import json
+
+from flask import current_app
+
+from ml_enabler.predictors import predictors
+from ml_enabler.aggregators import aggregators
+from ml_enabler.utils.api import get_model_id, post_prediction, post_prediction_tiles
+
+class MLEnablerService:
+
+    @staticmethod
+    def get_prediction_from_bbox(model_name:str, bbox: str, zoom: int=18):
+        """Sends requests to create a new prediction for a bounding box from a model id"""
+        api_url = current_app.config['ML_ENABLER_URL']
+        model_id = get_model_id(api_url, model_name)
+        url = f'{api_url}/model/{model_id}/tiles' 
+        params = dict(bbox=bbox, zoom=zoom)
+        response = requests.get(url, params=params)
```

Comment:
We should include after getting the response, a raise error depending on status. It is solved using _response.raise_for_status()_. This would raise an error depending on the status code received as response. Then, we catch that error with **requests.exceptions.RequestException** in _ml_enabler_apis.py_